### ISSUE TITLE
[Enhancement]: Support dark theme-styling in emails

### ISSUE THREAD

**Issue by barryhughes** [woothemes/woocommerce#32929]
### Describe the solution you'd like

A number of email clients support "dark mode" for their UI and can also render a dark mode version of emails, if the necessary styles are supplied using the `@media (prefers-color-scheme: dark)` media query. It would be great to support this.

### Describe alternatives you've considered

The alternative would be to not provide any sort of built-in support for dark theme emails.

### Additional context

See prior art:

- https://github.com/woocommerce/woocommerce/pull/27114
- https://github.com/woocommerce/woocommerce/pull/26744

**Comment by samrat-biswas-git** [comment]
We want to do this — dark-mode email support is worth having. Here's where it stands and what we need next.

  - No dark-mode support in core today (nothing using prefers-color-scheme).
  - It's been attempted twice — #26744 and #27114 — both closed unmerged, so it's known to be tricky: email-client support for prefers-color-scheme is patchy, and clients like
  Gmail and Outlook apply their own forced color inversion you can't fully override.
  - The natural home now is the block email editor's theming system (the central color palette / theme-sync). That should make a coherent dark variant far more feasible than
  the earlier one-off attempts.

  Next step is an engineering assessment: how feasible is a dark variant built on the block editor's color system, what's the rough effort, and what can we realistically
  control across the major email clients (vs. what we'd leave to client auto-inversion)?

  @barryhughes — if you have notes on what blocked #26744 / #27114, that'd give whoever picks this up a real head start.

**Comment by barryhughes** [comment]
> @barryhughes — if you have notes on what blocked https://github.com/woocommerce/woocommerce/pull/26744 / https://github.com/woocommerce/woocommerce/pull/27114, that'd give whoever picks this up a real head start.

I don't have any technical notes to share. Unfortunately, it took 18 months for #27114 to receive a reply and, by then, the author had presumably (and understandably) moved on. At the same time, our team at the time did not have bandwidth to take over the work. So, I'd say the field is wide open for a fresh attempt (and it may or may not make sense to base it on what you can see in #27114).

**Comment by samrat-biswas-git** [comment]
Makes sense. It's New-Feature work so it's behind the current baseline-quality push and so we will revisit it as soon as we are done with the current sweep of bugs.

**Comment by gigitux** [comment]
I’m going to remove the Kirigami label from this issue, as it doesn’t fall within this team’s area of responsibility.
